osu.Game/Screens/Play/HUD/SongProgress.cs

```diff
@@ -3,18 +3,31 @@
 
 using System;
 using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
 using osu.Framework.Allocation;
 using osu.Framework.Bindables;
+using osu.Framework.Extensions;
 using osu.Framework.Graphics;
 using osu.Framework.Graphics.Containers;
 using osu.Framework.Timing;
 using osu.Game.Beatmaps;
+using osu.Game.Rulesets;
+using osu.Game.Rulesets.Mods;
 using osu.Game.Rulesets.Objects;
 using osu.Game.Rulesets.UI;
 using osu.Game.Skinning;
 
 namespace osu.Game.Screens.Play.HUD
 {
+    public enum DifficultyGraphType
+    {
+        None,
+        ObjectDensity,
+        MaxStrain,
+        TotalStrain
+    }
```

Comment:
did anybody even ask for these options? this is too much

Reply:
`None` and `ObjectDensity` are ones that were before
`MaxStrain` and `TotalStrain` are new one
I think I will leave only `TotalStrain`